spec/paperclip/io_adapters/file_path_adapter_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.